aequilibrae/distribution/ipf.py

```diff
@@ -1,13 +1,21 @@
+import importlib.util as iutil
 import os
+from datetime import datetime
 from time import perf_counter
 from uuid import uuid4
-from datetime import datetime
-import importlib.util as iutil
+
 import numpy as np
 import yaml
+
+from aequilibrae import global_logger
+from aequilibrae.context import get_active_project
 from aequilibrae.matrix import AequilibraeMatrix, AequilibraeData
 from aequilibrae.project.data.matrix_record import MatrixRecord
-from aequilibrae.context import get_active_project
+
```

Comment:
@jamiecook and @djfrancesco Does any of you have any idea of why do I have to wrap this in a try/except in order for it to build on Linux and MacOS?

Reply:
It seems that the error originates from the `__init__.py` file in the aequilibrae subfolder. There is this import statement: `from aequilibrae import distribution` wether `compiled` is True or False. If we move it after `if compiled`, the try/except is not required anymore